### ISSUE TITLE
HMMER2 entry requested

### ISSUE THREAD

**Issue by smoe** [bio-tools/biotoolsRegistry#412]
Debian has a package for HMMER 2. There is a bio.tools entry hmmer3 only.

**Comment by joncison** [comment]
For now, I just created a boilerplate entry here:
https://bio.tools/HMMER2

pointing to the latest (and detailed) tool / description:
https://bio.tools/hmmer3

is this OK? cc @hansioan

**Comment by smoe** [comment]
Should it be hmmer2 - all lower case?
Debian had community requests to reintroduce hmmer2 after it had transitioned to hmmer3, so we thought there just should be some bio.tools entry to reference it. I just came across https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5126834/ "xHMMER3x2: Utilizing HMMER3’s speed and HMMER2’s sensitivity and specificity in the glocal alignment mode for improved large-scale protein domain annotation".

**Comment by joncison** [comment]
Interesting!   At least here people are using older tools but for good reasons.

At some point we'll provide entries for all the individual HMMER programs ... this would be much more useful than the current huge entry (which would be retained, but in a simpler form)

The IDs are case-insensitive (we periodically lower-caseify them).    So you should use https://bio.tools/hmmer2 ideally (but any case permutation should resolve - trying not to break the internet n'all that :)

**Comment by smoe** [comment]
https://salsa.debian.org/med-team/hmmer2/blob/master/debian/upstream/metadata
Updated.

To have the individual binaries as entries in bio.tools is interesting. This would allow for more detailed annotation by EDAM. But also means much more work. Sigh.

**Comment by joncison** [comment]
yeah ... it's sort of necessary, really.  We have it already for EMBOSS and SAMtools (if memory serves).  A good task for our student curators (cc @hansioan) - when we get them.
